client/src/components/main.jsx

```diff
@@ -11,28 +11,42 @@ class Main extends React.Component {
     }
   }
 
-  // componentDidMount() {
-  //   this.setState({details: this.props.details})
-  // }
-
   render () {
-    var test = this.props.details[1]
+    var test = this.props.details[0]
     return (
       console.log(this.props),
```

Comment:
@SimonC989 @carl12 Try to avoid leaving stray console logs in code that will be merged into master. Note that the react developer tools allow one to view a component's props and state in real time (could be easier than using a console.log)